src/project/formats/hal.ts

```diff
@@ -34,16 +34,47 @@ export function item(project: Project) {
       },
       collection: {
         title: 'List of projects',
-        href: '/project'
+        href: '/project',
       },
       client: {
         title: project.client.name,
         href: project.client.href,
       },
       'entry-collection': {
         title: 'List of time entries',
-        href: project.href + '/entry'
-      }
+        href: project.href + '/entry',
+      },
+    },
+    _templates: {
+      add: {
+        title: 'Add a Person to a Project',
+        method: 'POST',
+        contentType: 'application/json',
+        target: `${project.href}/person`,
+        properties: [
+          {
+            name: 'Role',
+            prompt: 'Role',
+            required: true,
+            options: {
+              inline: ['member', 'manager', 'owner'],
+            },
+          },
+          {
+            name: 'name',
+            prompt: 'Name',
+            required: true,
+            placeHolder: 'John Doe',
+          },
+          {
+            name: 'email',
+            prompt: 'Email',
```

Comment:
If we're going by the `mailto:` format, lets call this field `href` and prompt `Href`

Reply:
hey so when I change the type to 'mailto:' i get an error, I'm assuming it still has to be `url` ?